src/workerd/io/worker-interface.capnp

```diff
@@ -146,6 +146,7 @@ struct QueueMessage @0x944adb18c0352295 {
   timestampNs @1 :Int64;
   data @2 :Data;
   contentType @3 :Text;
+  attempts @4 :UInt16;
```

Comment:
If the answer to https://github.com/cloudflare/workerd/pull/1874/files#r1534621760 is that the first attempt of a message is meant to be delivered with an `attempts` of 1, then perhaps we should just default this field to 1 when it isn't explicitly set by the client? By doing:

```
  attempts @4 :UInt16 = 1;
```

Reply:
It must be always set by the client (we already made the changes), o/w we may be opening the door to bugs.